plugins/rebalance/rebalance.go

```diff
@@ -0,0 +1,109 @@
+package rebalance
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+	"os/exec"
+	"path"
+
+	"github.com/cespare/xxhash"
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	config "github.com/spf13/viper"
+)
+
+const (
+	// sbinDir represents bin directory
+	sbinDir = "glusterfs"
+)
+
+// RebalanceProcess type represents information about rebalance process
+type RebalanceProcess struct {
+	binarypath     string
+	args           string
+	socketfilepath string
+	pidfilepath    string
+	rebalance      RebalanceInfo
+}
+
+// Name returns the process name
+func (r *RebalanceProcess) Name() string {
+	return "rebalance"
+}
+
+// Path returns absolute path to the binary of rebalance process
+func (r *RebalanceProcess) Path() string {
+	return r.binarypath
+}
+
+// Args returns the arguments to be passed to rebalance process
+func (r *RebalanceProcess) Args() string {
+
+	shost, _, _ := net.SplitHostPort(config.GetString("clientaddress"))
+	if shost == "" {
+		shost = "localhost"
+	}
+
+	volFileID := path.Join("rebalance/", r.rebalance.Volname)
+	File := path.Join(config.GetString("logdir"), "glusterfs", r.rebalance.Volname)
+	logFile := fmt.Sprintf("%s-rebalance.log", File)
+	cmd := r.rebalance.Status
```

Comment:
This is requesting the rebalance status. Is this correct?

Reply:
yes. r.rebalance.Status will return Started